examples/asr/speech_to_text_bpe.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+python speech_to_text_bpe.py \
+    model.train_ds.manifest_filepath="./an4/train_manifest.json" \
+    model.validation_ds.manifest_filepath="./an4/test_manifest.json" \
+    model.tokenizer.path="./an4/tokenizer/LibriSpeechTokenizer/librispeech_tokenizer_bpe_v1024/" \
+    pl.trainer.gpus=2 \
+    pl.trainer.distributed_backend="ddp" \
+    pl.trainer.max_epochs=100 \
+    model.optim.name="adamw" \
+    model.optim.lr=0.1 \
+    model.optim.args.params.betas=[0.9,0.999] \
+    model.optim.args.params.weight_decay=0.0001 \
+    model.optim.sched.args.params.warmup_ratio=0.05 \
+    model.logger.experiment_name="AN4-BPE-1024" \
+    model.logger.project_name="AN4_BPE_1024_candidate" \
+    hydra.run.dir=.
+"""
+
+import hydra
+import pytorch_lightning as pl
+from pytorch_lightning.logging import WandbLogger
+
+from nemo.collections.asr.models.ctc_bpe_models import EncDecCTCModelBPE
+from nemo.utils import logging
+
+
+@hydra.main(config_path="experimental/configs/", config_name="config_bpe")
```

Comment:
but in config_bpe labels are chars! how does this work?

Reply:
Fixed